Hyperlint config updates

kodster28 opened the pull request:
### Summary

@bllchmbrs, question... does the `custom_comment` access `\n` characters? And/or, does it accept markdown input as well?

bllchmbrs commented:
@kodster28 , will take a look at this a bit later this AM and make sure you're all good! I *believe* the multi-line yaml should just work!

bllchmbrs commented:
@kodster28 , just copy and paste this:

```yaml
content_dir: docs
reviewer:
  custom_comment: |-
    Howdy and thanks for contributing to our repo. The Cloudflare team reviews new, external PRs within **two (2) weeks**. If it's been two weeks or longer without any movement, please tag the PR **Assignees** in a comment.

    We review internal PRs within **1 week**. If it's something urgent or has been sitting without a comment, start a thread in the *Developer Docs* space internally.
  enabled: true
  external_link_validation:
    check_status_upon_review_failure: neutral
  summarize_pull_request:
    enabled: true
  vale_style_guide:
    check_status_upon_review_failure: neutral
    enabled: true

```

That respects multi-line for your 'preamble' comment. I did edit it for a bit of clarity (old writing habits die hard 😉)

I also changed the check status to neutral on failure. It won't "bark" like the ❌ but it still provides a subtle note for the observant.

kodster28 commented:
Howd

> @kodster28 , just copy and paste this:
> 
> ```yaml
> content_dir: /src/content
> reviewer:
>   custom_comment: |-
>     Howdy and thanks for contributing to our repo. The Cloudflare team reviews new, external PRs within **two (2) weeks**. If it's been two weeks or longer without any movement, please tag the PR **Assignees** in a comment.
> 
>     We review internal PRs within **1 week**. If it's something urgent or has been sitting without a comment, start a thread in the *Developer Docs* space internally.
>   enabled: true
>   external_link_validation:
>     check_status_upon_review_failure: neutral
>   summarize_pull_request:
>     enabled: true
>   vale_style_guide:
>     check_status_upon_review_failure: neutral
>     enabled: true
> ```
> 
> That respects multi-line for your 'preamble' comment. I did edit it for a bit of clarity (old writing habits die hard 😉)
> 
> I also changed the check status to neutral on failure. It won't "bark" like the ❌ but it still provides a subtle note for the observant.

Okay, Adjusted but didn't know if we still needed the `openapi` + `authorized_users` separately

bllchmbrs commented:
you can leave them, or remove them, they are ignored at this point!
